addons/mail/static/src/scss/mail_notification.scss

```diff
@@ -0,0 +1,4 @@
+.o_mail_notification_failure_reason {
```

Comment:
Still this css I don't like ... actually why do we need it ? Especially in a view that is highly technical, should be as standard as possible...